executor.c

```diff
@@ -10,11 +10,13 @@
 
 int executor(char **args, char **env)
 {
-	int success = 0;
-	
-	success = execve(args[0], args, env);
-	if (success < 0)
-		return (0);
```

Comment:
don't return here, if u return the child process will return to the function that called executor and continue executing it, u'll have 2 simple shells instead of one, exit instead